hyp3_autorift/geometry.py

```diff
@@ -101,19 +102,18 @@ def polygon_from_bbox(lat_limits, lon_limits):
     return polygon
 
 
-def find_jpl_dem(lat_limits, lon_limits):
+def find_jpl_dem(polygon: ogr.Geometry) -> str:
     shape_file = f'/vsicurl/http://{ITS_LIVE_BUCKET}.s3.amazonaws.com/{AUTORIFT_PREFIX}/autorift_parameters.shp'
     driver = ogr.GetDriverByName('ESRI Shapefile')
     shapes = driver.Open(shape_file, gdal.GA_ReadOnly)
 
-    centroid = polygon_from_bbox(lat_limits, lon_limits).Centroid()
+    centroid = polygon.Centroid()
     for feature in shapes.GetLayer(0):
         if feature.geometry().Contains(centroid):
             return f'{feature["name"]}_0240m'
 
     raise DemError('Could not determine appropriate DEM for:\n'
-                   f'    lat (min, max): {lat_limits}'
-                   f'    lon (min, max): {lon_limits}'
+                   f'    centroid: {centroid}'
```

Comment:
I don't think this has a useful `str` representation, may have to be explicit and print `centroid.GetX()`, `centroid.GetY()`

Reply:
String representation is actually decent:

```python
>>> polygon = polygon_from_bbox(lat_limits=(55, 56), lon_limits=(40, 41))
>>> centroid = polygon.Centroid()
>>> print(f'centroid: {centroid}')
centroid: POINT (40.5 55.5)
```